packages/terra-card/src/Card.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import './Card.scss';
+
+const propTypes = {
+  /**
+   * Provides 10px on top and bottom
+   */
+  padVertical: PropTypes.bool,
+  /**
+   * Provides 10px on left and right
+   */
+  padHorizontal: PropTypes.bool,
+  /**
+   * Child Nodes
+   */
+  children: PropTypes.node,
+};
+
+const defaultProps = {
+  padVertical: true,
+  padHorizontal: true,
+};
+
+const Card = ({
+  padVertical,
+  padHorizontal,
+  children,
+  ...customProps
+}) => {
+  // Follows these conventions https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties
```

Comment:
This comment is no longer applicable as we are just doing the 2 properties instead of the css shorthand style array of values